prediction/src/preprocess/improved_lung_segmentation.py

```diff
@@ -0,0 +1,444 @@
+import numpy as np
+import scipy.ndimage
+import skimage
+import skimage.measure
```

Comment:
If you import `skimage` you can already access its submodules like `measures` by `skimage.measure` in the code; you're not supposed to explicitly import `skimage.measure` again :) Same applies for importing filters, morphology and transform for skimage and scipy.ndimage.morphology for scipy.

Reply:
I'm not a python master, but it doesn't work without explicit imports (:

Reply:
Curiously this is not always the case.. Python imports are *weird* :)